src/commands/fix_lock_files/mod.rs

```diff
@@ -34,13 +37,19 @@ pub struct Options {
 /// dependencies slip into CI.
 ///
 /// Any workspaces containing a ".no_cargo_lock" sentinel file will be skipped.
-pub fn fix_workspace_lockfile(
-    repo: &Repository,
+pub async fn fix_workspace_lockfile(
```

Comment:
Why are you making it `async` if you never use `await` in the body?

Reply:
It's misleading to the consumer and harder to use. Any async function requires a runtime, but not all contexts are async or have a runtime available.